posthog/api/cohort.py

```diff
@@ -58,6 +59,7 @@
 class CohortSerializer(serializers.ModelSerializer):
     created_by = UserBasicSerializer(read_only=True)
     earliest_timestamp_func = get_earliest_timestamp
+    # Note: This nullifies validated_data payload on create/update
```

Comment:
Serializermethodfield is [read-only](https://www.django-rest-framework.org/api-guide/fields/#serializermethodfield). As a result, we need to make sure to manually parse this field during a create/update.

Reply:
@neilkakkar added a solution below to just handle the parsing otherwise we could change below to `properties = serializers.SerializerMethodField()` or something else so it doesn't conflict with filters?

A custom serializer doesn't quite work I think because we want to access cohort.properties